nominees/casa-volunteer-tracker.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "CASA Volunteer Tracker",
+  "aliases": [
+    "CASA"
+  ],
+  "description": "Volunteer management system for nonprofit CASA, which serves foster youth in counties across America.",
+  "website": "https://www.casavolunteertracking.org/",
```

Comment:
Please request submitter to share a link that provides more information about the digital solution as the [current one](https://www.casavolunteertracking.org/) simply redirects to the password protected platform. Thanks.

Reply:
CASA Volunteer Tracker owner responded:
> You can find more information about the project here: https://github.com/rubyforgood/casa#readme

Alternatives found on the Ruby for Good that mention this project could also be considered:
- https://rubyforgood.org/our-work/impact
- https://pgcasa.org/